RubberduckTests/Refactoring/EncapsulateFieldTests.cs

```diff
@@ -1627,6 +1629,10 @@ private static Mock<IRefactoringPresenterFactory<IEncapsulateFieldPresenter>> Se
             return factory;
         }
 
+        private static IIndenter CreateIndenter(VBE vbe)
+        {
+            return new Indenter(vbe, () => new IndenterSettings());
```

Comment:
IndenterSettings needs to be mocked - the ctor checks the registry.  IndenterSettingsTests.GetMockIndenterSettings is public though.